microsoft/PowerToys
# Standardize .NET JSON on System.Text.Json

## Summary of the Pull Request

**What is this about:**
Replaces all (direct) references of Newtonsoft.Json with System.Text.Json

**What is include in the PR:** 

**How does someone test / validate:** 
- Run all tests
- Check if settings are correctly applied

## Quality Checklist

- [x] **Linked issue:** #1675

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
